src/components/buttons/AddButton.js

```diff
@@ -0,0 +1,31 @@
+import styled from 'styled-components';
+import { Button } from 'lattice-ui-kit';
+
+const BG_1 = '#d3f0e6'; // default
```

Comment:
Can we add these colors and button variations to LUK? `success` and `danger` perhaps?